.github/workflows/build-git-installers.yml

```diff
@@ -797,7 +797,7 @@ jobs:
 
             // Create the release
             var tagName = "${{ needs.prereqs.outputs.tag_name }}";
-            var createdRelease = await github.repos.createRelease({
+            var createdRelease = await github.rest.repos.createRelease({
```

Comment:
I hope you did not have a hard time finding out about that; I should have remembered to point out that I had to fix plenty of those [in the corresponding git/git patch](https://lore.kernel.org/git/472481561f03900fb519065d88ae8af910bea794.1670423680.git.gitgitgadget@gmail.com/).